src/libsyntax/config.rs

```diff
@@ -157,11 +157,13 @@ impl<'a> StripUnconfigured<'a> {
         // flag the offending attributes
         for attr in attrs.iter() {
             if !self.features.map(|features| features.stmt_expr_attributes).unwrap_or(true) {
-                emit_feature_err(&self.sess,
-                                 "stmt_expr_attributes",
-                                 attr.span,
-                                 GateIssue::Language,
-                                 EXPLAIN_STMT_ATTR_SYNTAX);
+                feature_err(&self.sess,
+                            "stmt_expr_attributes",
+                            attr.span,
+                            GateIssue::Language,
+                            EXPLAIN_STMT_ATTR_SYNTAX)
+                    .help("`///` is for documentation comments. For a plain comment, use `//`.")
```

Comment:
Oh, I forgot to bring this up again.

This help should only be generated for misplaced doc comments, but this will generate it for any misplaced attributes at all.